x/allocation/types/params.go

```diff
@@ -0,0 +1,134 @@
+package types
+
+import (
+	"errors"
+	fmt "fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+// Parameter store keys
+var (
+	KeyDistributionProportions   = []byte("DistributionProportions")
+	KeyDeveloperRewardsReceiver  = []byte("DeveloperRewardsReceiver")
+	KeyRandomnessRewardsReceiver = []byte("RandomnessRewardsReceiver")
+)
+
+// DefaultParams module parameters
+func DefaultParams() Params {
+	return Params{
+		DistributionProportions: DistributionProportions{
+			RandomnessRewards: sdk.NewDecWithPrec(4, 2),  // 4%
+			ValidatorRewards:  sdk.NewDecWithPrec(6, 2),  // 6%
+			DeveloperRewards:  sdk.NewDecWithPrec(20, 2), // 20 %
+		},
+		WeightedDeveloperRewardsReceivers: []WeightedAddress{},
```

Comment:
Could you add `randomness_rewards_receiver` for visibility?

Reply:
added